libs/perun/components/src/lib/report-issue-dialog/report-issue-dialog.component.ts

```diff
@@ -2,11 +2,11 @@ import { Component, OnInit } from '@angular/core';
 import { MatDialogRef } from '@angular/material/dialog';
 import { TranslateService } from '@ngx-translate/core';
 import { NotificatorService, StoreService } from '@perun-web-apps/perun/services';
-import { version } from '../../../../../../../package.json';
+import { version } from '../../../../../../package.json';
 import { RTMessagesManagerService } from '@perun-web-apps/perun/openapi';
 
 @Component({
-  selector: 'app-perun-web-apps-report-issue-dialog',
+  selector: 'perun-web-apps-report-issue-dialog',
   templateUrl: './report-issue-dialog.component.html',
   styleUrls: ['./report-issue-dialog.component.scss']
```

Comment:
translation for report issue dialog not working 
![image](https://user-images.githubusercontent.com/32871167/110105697-00206480-7da9-11eb-99b1-428b969a3546.png)